Streetcode/Streetcode.BLL/MediatR/Streetcode/Streetcode/Handlers/QueryHandlers/GetStreetcodeById.cs

```diff
@@ -0,0 +1,36 @@
+﻿using AutoMapper;
```

Comment:
Also please consider to switch onto the file structure which was provided in my example (Folder per action, Query/Command and Handler go inside of it).

Reply:
done